bin/tests/common.rs

```diff
@@ -153,7 +152,18 @@ impl<'a> AgentSettings<'a> {
 }
 
 pub fn spawn_agent(settings: AgentSettings) -> Child {
-    let mut cmd = Command::cargo_bin("logdna-agent").unwrap();
```

Comment:
Using escargot instead of just assert_cmd's cargo support gives us a lot more control over the integration test build, specifically it allows us to select the optimisation level of the target